stable/external-dns/Chart.yaml

```
@@ -3,7 +3,7 @@ description:
   Configure external DNS servers (AWS Route53, Google CloudDNS and others)
   for Kubernetes Ingresses and Services
 name: external-dns
-version: 1.3.3
+version: 1.3.4
```

adding new funcionality 
```suggestion
version: 1.4.0
```